tools/monorepo-merge/src/commands/transfer-issues/index.ts

```diff
@@ -345,22 +362,27 @@ export default class TransferIssues extends Command {
 	 * Gets all of the issues that we are going to transfer into the monorepo.
 	 *
 	 * @param {graphql} authenticatedGraphQL The graphql object for making requests.
-	 * @param {string} source The repository to transfer issues from.
-	 * @param {string} searchFilter The GitHub search filters for the issues to transfer.
+	 * @param {string}  source               The repository to transfer issues from.
+	 * @param {string}  searchFilter         The GitHub search filters for the issues to transfer.
 	 */
 	private async getIssues(
 		authenticatedGraphQL: typeof graphql,
 		source: string,
 		searchFilter: string
-	): Promise< GitHubIssue[] > {
+	) {
```

Comment:
I just want to mention that I think removing return type signatures is a good idea. The return type will always be inferred so if you change something in the implementation of this function it will complain elsewhere in your code.

Before even though the type of the return value was `any` in many cases it did not complain because of the explicit return type.